rtc-video-quality/generate_data.py

```diff
@@ -349,9 +349,11 @@ def run_command(job, encoder_command, job_temp_dir, encoded_file_dir):
     clip = job['clip']
     start_time = time.time()
     try:
-        process = subprocess.Popen(command,
+        process = subprocess.Popen(' '.join(command),
                                    stdout=subprocess.PIPE,
-                                   stderr=subprocess.STDOUT)
+                                   stderr=subprocess.STDOUT,
+                                   encoding='utf-8',
+                                   shell=True)
```

Comment:
In general, it's better to avoid using shell=True because of the possibility of shell injection vulnerabilities.
https://docs.python.org/3.6/library/subprocess.html#security-considerations

That said, in this case it's clearly the simplest solution, and this code isn't intended to be used in any untrusted environments. Given our limited time, I'm okay with using it here.

Reply:
Yeah this did come to my mind when I was searching up for`shell=True`. But I am assuming we won't be requesting input from the user so that we won't be exposed to this vulnerability. But it is important to keep in mind.  

The other approach I was thinking was to have nested lists and run subprocess.Popen in a for loop. But that was a bit of a hassle as it involved changing a lot of other areas in the code

Reply:
Agreed. I just wanted to point it out, but I think in this case it's worth using shell=True.